app/models/katello/concerns/organization_extensions.rb

```diff
@@ -203,6 +203,16 @@ def notification_recipients_ids
           users.pluck(:id)
         end
 
+        def audit_manifest_action(message)
```

Comment:
If the existing tests for this concern make it straightforward I'd like to see a test for this method

One other thing to consider is that if there is a possibility the method signature should change it might be beneficial to use hash params, but I'm OK with the way it is too.

Reply:
test added!

I don't see a huge need for hash parameters right now, but feel free to share an example as I may be missing something.

Reply:
I was thinking of something like this:

`audit_manifest_action(message: 'my message', new_param: 'some other value')` but in the small scope of this method it doesn't buy much - just wanted to see if you saw any value in it